designs/administrator/olm/operatorhub-features-subscription-filters/index.md

```diff
@@ -3,18 +3,19 @@ parent: Administrator
 version: 4.5
 ---
 
-# OperatorHub 'Infrastructure Features' and 'Paid Subscription Required' Filters
+# OperatorHub 'Infrastructure Features' and 'Valid Subscriptions'
 
-OLM provides a list of attributes that the operator developer can tag their operator with (or the operator is tagged with automatically), that describe its infastructure features, for example ‘disconnected’, ‘FIPS mode’, etc. This list is dictated by OLM as far as what attributes are available. There is also an attribute that designates which subscription (if any) the operator requires.
+OLM provides a list of attributes that the operator developer can tag their operator with (or the operator is tagged with automatically), that describe its infastructure features, including ‘disconnected’, ‘FIPS mode’, and 'proxy'. This list is dictated by OLM as far as what attributes are available. There is also an attribute that designates which subscriptions (if any) the operator requires.
 
-These new attributes should be surfaced in OperatorHub so that the admin can filter by them, as well as displayed on the Operators install page, along with the existing attributes, for awareness.
+These new feature attributes should be surfaced in OperatorHub so that the admin can filter by them, as well as displayed on the Operators install page, along with the existing attributes, for awareness. Because a user likely wouldn't filter based on an operator requiring a subscrption (and not operators they already have a subscription to,) this attribute will just be surfaced in the operator install details for now.
```

Comment:
```suggestion
These new feature attributes should be surfaced in OperatorHub so that the admin can filter by them, as well as displayed on the Operators install page, along with the existing attributes, for awareness. Because a user likely wouldn't filter based on an operator requiring a subscription (and not operators they already have a subscription to), this attribute will just be surfaced in the operator install details for now.
```